extensions/widgets/trending/src/components/cards/latest-topics.tsx

```diff
@@ -247,7 +247,7 @@ export const LatestTopics: React.FC<LatestTopicsProps> = props => {
     }
 
     // if subscribing, and limit is already reached, return snackbar
-    if (subscribing && localSubscribedTagsRef.current.length === 10) {
+    if (subscribing && localSubscribedTagsRef.current.length >= 10) {
```

Comment:
Unrelated to the changes in this PR, I've took a look at the overall business logic for the topic subscription, it looks very bad, unreadable and overly complicated